views/index.ejs

```diff
@@ -1,11 +0,0 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title><%= title %></title>
-    <link rel='stylesheet' href='/stylesheets/style.css' />
```

Comment:
Make sure that your style.css is accessible here. You will need to implement this by putting into public folder and then using static in express.